packages/gis/lua/TerraLib.lua

```diff
@@ -813,6 +813,10 @@ local function createDataSetAdapted(dSet, missing)
 			local type = dSet:getPropertyDataType(i)
 
 			if dSet:isNull(i) then
+				if type == binding.GEOMETRY_TYPE then
+					customError("Data cannot be used, because it has missing geometry. Please, fix your data.")
```

Comment:
"Data cannot be loaded because it has a missing geometry. Please, fix your data."